include/envoy/stats/scope.h

```diff
@@ -19,9 +19,9 @@ class Histogram;
 class Scope;
 class NullGaugeImpl;
 
-using OptionalCounter = absl::optional<std::reference_wrapper<const Counter>>;
-using OptionalGauge = absl::optional<std::reference_wrapper<const Gauge>>;
-using OptionalHistogram = absl::optional<std::reference_wrapper<const Histogram>>;
+using CounterOptRef = absl::optional<std::reference_wrapper<const Counter>>;
```

Comment:
should this be CounterOptConstRef per your style section?

Reply:
Yeah, I should definitely follow my rules 😅. I checked all the other aliases for correctness.

Reply:
Overall I would like the aliases to be CI checked in our formatting run. I will open an issue for that if someone wants to pick up

Reply:
https://github.com/envoyproxy/envoy/issues/9840